ludwig/models/modules/convolutional_modules.py

```diff
@@ -519,6 +524,349 @@ def call(self, inputs, training=None, mask=None):
         return hidden
 
 
+class Conv2DLayerFixedPadding(Layer):
+
+    def __init__(
+            self,
+            num_filters=256,
+            filter_size=3,
+            strides=1,
+            weights_regularizer=None
+    ):
+        super(Conv2DLayerFixedPadding, self).__init__()
+
+        self.layers = []
+
+        if strides > 1:
+            self.layers.append(ZeroPadding2D(padding=((filter_size - 1) // 2)))
```

Comment:
@w4nderlust have a look at this one, does this padding size make sense compared to the previous implementation?